resources/js/components/layouts/Header.js

```diff
@@ -38,14 +38,13 @@ const Header = (props) => {
     // Get the cookie activePage value
     const uriName = props.cookies.get("activePage");
     if (uriName !== undefined) {
-      if (uriName === "/dashboard") {
-        setURI(uriName);
-      }
-      if (uriName === "/categories") {
-        setURI(uriName);
-      }
-      if (uriName === "/user_lists") {
-        setURI(uriName);
+      switch (uriName) {
+        case "/dashboard":
+          setURI(uriName);
+        case "/categories":
```

Comment:
here also https://www.w3schools.com/js/js_switch.asp

Reply:
This is resolved ma'am.